.github/WORKFLOW_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Hmm, this doesn't really follow GitHub issue template syntax... i.e. it's supposed to be a `.yml` file?

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

Reply:
`.yml` templates are available for users to create issues with, which is not what we want here. The documentation for https://github.com/JasonEtco/create-an-issue recommends using a .md file in this case. I can try changing this to a .yml file and see if I can exclude it from the create-an-issue page if you feel strongly about this!

Reply:
I just feel strongly about not confusing this with a GitHub issue template. You prob don't need to capitalize the whole name. Suggestion - rename to `.github/create_issue_on_error.md`

Reply:
Done!